### PR TITLE
Added: EVR parser function and test

### PR DESCRIPTION
**Pull request by ghost**
Did some basic testing with it - reject it if you want better tests.

### CODE REVIEW

File: src/version.rs [hunk 1]
```diff
@@ -0,0 +1,82 @@
+//split EVR into release, version and release
```
Comment (dns2utf8) [inline src/version.rs:1]: Why release twice?
Would you agree to use a struct?

**Review by dns2utf8 — CHANGES_REQUESTED** [review]
Where does `mod version` point? Are you planing to move it into that module?

It is a good start. If the return were a struct we could implement `PartialOrd` later

File: src/version.rs (hunk 1 above)
Comment (ghost) [reply src/version.rs:1]: It should have been 'epoche, version and release', my mistake - Ill change it.

For the return type? Should be fine in that case.
Should I use parse_evr() as the constructor for the struct?

**Comment by ghost** [comment]
'mod version' points to version.rs - I was planning to keep this and its related functions in this file, for now.

**Comment by ghost** [comment]
I don't mean to be intrusive but I sincerely appreciate a response on my statement.

**Pull request closed without merging**